### PR TITLE
CompressionType and useIdiomaticEndpoints for Protobuf and Avro

### PR DESCRIPTION
**Pull request by fedefernandez**
Relates to #85

### CODE REVIEW

**Comment by fedefernandez** [comment]
@franciscodr FYI

File: src/main/scala/higherkindness/skeuomorph/mu/print.scala [hunk 1]
```diff
@@ -136,6 +136,13 @@ object print {
       case Identity => Right(Identity)
     })
 
+  def idiomaticEndpoints: Printer[IdiomaticEndpoints] =
+    Printer {
+      case IdiomaticEndpoints(Some(pkg), true) => ", namespace = Some(\"" + pkg + "\"), methodNameStyle = Capitalize"
```
Comment (AlvaroCaste) [inline src/main/scala/higherkindness/skeuomorph/mu/print.scala]: out of curiosity: is there no "default" Printer for `Some` or `Option`?

Comment (fedefernandez) [reply src/main/scala/higherkindness/skeuomorph/mu/print.scala]: @AlvaroCaste I think I don't understand you, sorry. Could you please rephrase it?

Comment (fedefernandez) [reply src/main/scala/higherkindness/skeuomorph/mu/print.scala]: Oh, you mean there is no case for `Some` + `false`? Fixed!

Comment (AlvaroCaste) [reply src/main/scala/higherkindness/skeuomorph/mu/print.scala]: no, sorry, I got it wrong. I thought you were matching an `Option` and then I wondered if there would already be a `Printer` for it in order not to rewrite `"Some(\"" + pkg + "\")` (this happens when I've just skimmed it 😅 )

**Comment by fedefernandez** [comment]
Ready for review

**Pull request merged**